pkg/supportbundle/defaultspec/spec.yaml

```diff
@@ -95,6 +96,7 @@ spec:
         namespace: kurl
     - secret:
         collectorName: kotsadm-replicated-registry
+        name: kotsadm-replicated-registry # TODO: what is the correct name here?
```

Comment:
this is the correct name for the secret used by kotsadm.  app secret names are generated dynamically

Reply:
i clarified this comment